project4/styles.css

```
@@ -0,0 +1,362 @@
+* {
+    box-sizing: border-box;
+}
+
+@font-face {
+    font-family: 'Misto';
+
+    src: url('fonts/Misto.woff') format('woff'),
+
+}
+
+body {
+    margin: 0 auto;
+    background-color: #fff7f7;
+}
+
+button {
+    margin-left: auto;
+    margin-right: auto;
+}
+
+fieldset {
+    display: inline-block;
+    vertical-align: top;
+    width: 30%;
+    border: 2px solid #2e2d2d;
+}
+
+footer {
+    background-color: #000000;
+}
+
+form {
+    text-align: center;
+}
+
+h2 {
+    font-family: 'Misto', serif;
+    font-weight: normal;
+    font-size: 40px;
+    margin-left: 10px;
+}
+
+h3 {
+    color: #2e2d2d;
+    font-family: 'Lato', 'sans-serif';
```

The fallback default font for your webfont shouldn't be wrapped up in quotes:
```suggestion
    font-family: 'Lato', sans-serif;
```